integration_tests/cli/model-validation-cases.sh

```diff
@@ -508,3 +511,83 @@ result=$(echo "test1234" | dcld tx model update-model-version --vid=$vid_1 --pid
+check_response_and_report "$result" "\"otaChecksumType\": 1"
+check_response_and_report "$result" "\"releaseNotesUrl\": \"https://release.notes.url.info\""
+check_response_and_report "$result" "\"maxApplicableSoftwareVersion\": 32"
+check_response_and_report "$result" "\"minApplicableSoftwareVersion\": 5"
+check_response_and_report "$result" "\"specificationVersion\": 0"
+
+test_divider
+
+sv_1=$RANDOM
+echo "Create a Device Model Version with all fields with VID: $vid_1 PID: $pid_1 SV: $sv_1"
+result=$(echo 'test1234' | dcld tx model add-model-version --vid=$vid_1 --pid=$pid_1 --softwareVersion=$sv_1 \
+--softwareVersionString="1.0" --cdVersionNumber=21334 \
+--firmwareInformation="123456789012345678901234567890123456789012345678901234567890123" \
+--softwareVersionValid=true --otaURL="https://ota.url.info" --otaFileSize=123456789 \
+--specificationVersion=33 --otaChecksum="SGVsbG8gd29ybGQh" --releaseNotesURL="https://release.notes.url.info" \
+--otaChecksumType=1 --maxApplicableSoftwareVersion=32 --minApplicableSoftwareVersion=5   --from=$vendor_account_1 --yes)
+result=$(get_txn_result "$result")
+echo "$result"
+check_response_and_report "$result" "\"code\": 0"
+
+test_divider
+
+# Update the model version with Specification Version must fail
+echo "Update the model version with Specification Version must fail for VID: $vid_1 PID: $pid_1 SV: $sv_1"
+result=$(echo "test1234" | dcld tx model update-model-version --vid=$vid_1 --pid=$pid_1 --softwareVersion=$sv_1 --specificationVersion=11 --from=$vendor_account_1 --yes 2>&1) || true
+
+check_response "$result" "Error: unknown flag: --specificationVersion" raw
+
+test_divider
+
+# Query the model version 
+echo "Query Device Model Version with VID: $vid_1 PID: $pid_1 SV: $sv_1"
+result=$(dcld query model model-version --vid=$vid_1 --pid=$pid_1 --softwareVersion=$sv_1)
+
+check_response_and_report "$result" "\"vid\": $vid_1"
+check_response_and_report "$result" "\"pid\": $pid_1"
+check_response_and_report "$result" "\"softwareVersion\": $sv_1"
+check_response_and_report "$result" "\"softwareVersionString\": \"1.0\""
+check_response_and_report "$result" "\"cdVersionNumber\": 21334"
+check_response_and_report "$result" "\"firmwareInformation\": \"123456789012345678901234567890123456789012345678901234567890123\""
+check_response_and_report "$result" "\"softwareVersionValid\": true"
+check_response_and_report "$result" "\"otaUrl\": \"https://ota.url.info\""
+check_response_and_report "$result" "\"otaFileSize\": \"123456789\""
+check_response_and_report "$result" "\"otaChecksum\": \"SGVsbG8gd29ybGQh\""
+check_response_and_report "$result" "\"otaChecksumType\": 1"
+check_response_and_report "$result" "\"releaseNotesUrl\": \"https://release.notes.url.info\""
+check_response_and_report "$result" "\"maxApplicableSoftwareVersion\": 32"
+check_response_and_report "$result" "\"minApplicableSoftwareVersion\": 5"
+check_response_and_report "$result" "\"specificationVersion\": 33"
```

Comment:
It's absolutely non-critical, but I suggest to use constant values for these cases instead of in-line numbers